bundles/org.openhab.binding.enturno/README.md

```diff
@@ -0,0 +1,96 @@
+# EnturNo binding
+
+This binding is meant to get Norwegian public transport real-time (estimated) data from [Entur.no API](https://developer.entur.org/content/journey-planner-0).
+                                                        
+## Supported Things
+
+As for now, binding supports only one thing. It can change in the future as entur.no exposes API for access of different type 
```

Comment:
Can you put all single lines in the readme on a new line. This makes review changes (in the future) more easier.